platforms/xamarin.ios/System.Runtime.InteropServices.RuntimeInformation.cs

```diff
@@ -0,0 +1,32 @@
+namespace System.Runtime.InteropServices
```

Comment:
@weshaggard ok, I was going by your comment in https://github.com/dotnet/standard/pull/31#issuecomment-251309088 which included those APIs. I'll remove it.